crates/binstalk/src/ops/resolve.rs

```diff
@@ -83,10 +83,6 @@ async fn resolve_inner(
         return Ok(Resolution::AlreadyUpToDate);
     };
 
-    if opts.signature_policy == SignaturePolicy::Require && !package_info.signing {
-        return Err(BinstallError::MissingSignature(package_info.name));
-    }
-
```

Comment:
We can't do that anymore, as if quickinstall has a signature we can't see that until we run the find().